### PR TITLE
Bugfix in SphereFitting and TensorVoting

### PR DESCRIPTION
**Pull request by dcoeurjo**

### CODE REVIEW

File: src/DGtal/geometry/surfaces/estimation/estimationFunctors/SphereFittingEstimator.h [hunk 1]
```diff
@@ -68,8 +68,11 @@ namespace DGtal
      * Description of template class 'SphereFittingEstimator' <p>
      * \brief Aim: Use Patate library to perform a local sphere fitting.
      *
-     * model of CLocalEstimatorFromSurfelFunctor.
+     * Given a kernel radius, this functor performs a sphere fitting
+     * and outputs the paramters of an AlgebraicSphere (please see
```
Comment (kerautret) [inline src/DGtal/geometry/surfaces/estimation/estimationFunctors/SphereFittingEstimator.h]: paramEters

**Comment by kerautret** [comment]
Just see a typo but you can merge...

**Comment by dcoeurjo** [comment]
thanks, done

**Comment by kerautret** [comment]
ok merging

**Pull request merged**